### ISSUE TITLE
Rollup bundle error with Typescript

### ISSUE THREAD

**Issue by childrentime** [tj/commander.js#1781]
```ts
import { Command } from "commander";
```

when use rollup to bundle it, it throw

```sh
[!] Error: 'default' is not exported by node_modules/commander/index.js, imported by node_modules/commander/esm.mjs
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
node_modules/commander/esm.mjs (1:7)
1: import commander from './index.js';
```

change it to will fix, but we lose types

```ts
const { Command } = require("commander");
```

**Comment by shadowspawn** [comment]
I reproduced the problem using rollup with a simple esm file. I haven't tried reproducing using TypeScript. The problem is rollup finding the Commander code.

rollup works with esm modules, but Commander is written as CommonJS with an esm wrapper. The esm wrapper (`esm.mjs`) uses a default-style import which works with node.

https://nodejs.org/dist/latest-v16.x/docs/api/packages.html#approach-1-use-an-es-module-wrapper

This rollup issue covers related problems: https://github.com/rollup/rollup/issues/3514

I wonder if a configuration file could tell rollup where to find the Commander code? (Today was the first time I have looked at rollup, so this is only a guess!)
- https://rollupjs.org/guide/en/#configuration-files
